controller/argument_value_resolver.rst

```diff
@@ -148,14 +148,14 @@ and adding a priority.
 
     .. code-block:: yaml
 
-        # app/config/services.yml
+        # app/config/services.yaml
```

Comment:
`config/*` everywhere?

Reply:
Done for all services and routes files, the remaining onces need all be visited manually.